### PR TITLE
refactor(ErrorFormatter): FRMW-414-convert to Quark service.

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-414

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `rxStatus.ErrorFormatter` to a Quark service.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `ErrorFormatter` Quark Service

### Demo Page
screen shot 2015-12-30 at 7 51 01 pm

### API Docs
screen shot 2015-12-30 at 7 52 49 pm

**Comment by glynnis** [comment]
Design LGTM

File: src/quarks/ErrorFormatter/docs/examples/ErrorFormatter.simple.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div ng-controller="ErrorFormatterSimpleCtrl">
+  <p>
+      <button class="button"
```
Comment (CITguy) [inline src/quarks/ErrorFormatter/docs/examples/ErrorFormatter.simple.html]: Please correct indentation.

File: src/quarks/ErrorFormatter/docs/examples/ErrorFormatter.simple.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+angular.module('demoApp')
+.controller('ErrorFormatterSimpleCtrl', function ($scope, ErrorFormatter) {
+    $scope.setErrorMsg = function (msg) {
+        var error = { message: msg };
+
+		$scope.errorMsg = ErrorFormatter.buildErrorMsg('Error: ${message}', error);
```
Comment (CITguy) [inline src/quarks/ErrorFormatter/docs/examples/ErrorFormatter.simple.js]: Please correct indentation.

**Comment by 100stacks** [comment]
@CITguy updated...appreciate your feedback.  :+1:

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Comment by CITguy** [comment]
Considering that this (and other PRs) are getting caught up on `quarks.module.js`, and the addition to that file is minimal, please update this PR to remove changes made on `quarks.module.js` and we'll revisit documentation updates after Quarks are migrated.

**Pull request merged**